analysis/applySelections.cpp

```diff
@@ -294,13 +298,15 @@ int main(int argc, char* argv[])
       if(flag == "TauTau"){
         if(IsGoodForSingleMuon(*event))       events->AddEventToOutputTree(iEvent, outFilePaths[0], storeHLTtrees);
         if(IsGoodForMuEle(*event))            events->AddEventToOutputTree(iEvent, outFilePaths[1], storeHLTtrees);
+       // cout<<"test3"<<endl;
         if(IsGoodForMuMu(*event))             events->AddEventToOutputTree(iEvent, outFilePaths[2], storeHLTtrees);
+       // cout<<"test2"<<endl;
       }
     }
   }
   
   cout<<"Saving output trees"<<endl;
   for(string outFilePath : outFilePaths) events->SaveOutputTree(outFilePath);
-  
+  cout<<"test4"<<endl; 
```

Comment:
Please remove this cout, otherwise it will print a million times when we run on the full sample